src/pathfinding_service/model/token_network.py

```diff
@@ -190,22 +190,23 @@ def _check_validity(self) -> bool:
                     node=node,
                     node_reachability=node_reachability,
                 )
-                return False
+                return None
 
         # Calculate fees
         # This implicitely checks that the channels have sufficient capacity
-        valid_fees = self._calculate_fees()
-        if not valid_fees:
+        fees = self._calculate_fees()
+        if not fees:
             log.debug("Path invalid because of invalid fee calculation",)
-            return False
+            return None
 
-        return True
+        return fees
```

Comment:
If we moved the `log` to L132, we could directly do
```python
return self._calculate_fees()
```
This would also allow adding more details to the log, if necessary.